src/App.js

```diff
@@ -71,16 +77,15 @@ class App extends Component {
           </Switch>
         </BrowserRouter>
       );
-    }
-    if (!user.is_admin) {
+    } else {
```

Comment:
Tried to clean code: 'if (!user.is_admin)' is the same as 'else'

Reply:
thanks